recipe/meta.yaml

```diff
@@ -19,50 +19,52 @@ build:
 
 requirements:
   build:
+    - openssl 1.0.*                # [unix]
+    - curl >=7.44.0,<8
+    - cmake
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
     - python
-    - toolchain
     - pkgconfig                    # [unix]
     # For sha256 comparisons of opencv_contrib
-    - openssl 1.0.*                # [unix]
     # For downloading opencv_contrib
-    - curl >=7.44.0,<8
     # For applying patches
     - git                          # [win]
-    - cmake
-    - numpy 1.8.*   # [not (win and (py35 or py36))]
-    - numpy 1.9.*   # [win and py35]
-    - numpy 1.11.*  # [win and py36]
-    - hdf5 1.10.1         # [unix]
+    - numpy         # [not (win and (py35 or py36))]
+    - numpy         # [win and py35]
+    - numpy         # [win and py36]
```

Comment:
This is rather strange. Seems to have kept all the selectors even though it is switching to just `numpy`. Any ideas as to why this is?

cc @isuruf @CJ-Wright

Reply:
It would need more complicated logic to figure out that the 3 selector above can be reduced to `#[True]`